movie-app-code-review-main/src/app/components/img/img.ts

```diff
@@ -0,0 +1,26 @@
+import { div } from '@components/tags';
+
+import styles from './img.module.scss';
+
+interface Props {
+  src?: string;
+  alt?: string;
+  className?: string;
+}
+
+export const ImageWithPlaceholder = ({ src = '', alt = '', className = '' }: Props) => {
+  const image = new Image();
+  const wrapper = div(
+    {
+      className: styles.placeholder,
+    },
+    image,
```

Comment:
Suggestion for improvement. You can write this way, but it will be better if you use DOM.
const wrapper = document.createElement('div');
wrapper.classList.add(styles.placeholder);
wrapper.append(image);